packages/grid/x-data-grid/src/tests/selection.DataGrid.test.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-// @ts-ignore Remove once the test utils are typed
+// @ts-expect-error Remove once the test utils are typed
```

Comment:
Why?

Reply:
Once the test utils are typed, the type checker will automatically throw an error here. This way people know what to TODOs to remove once the issue is fixed. Some more info [here](https://github.com/microsoft/TypeScript/issues/37458)

Reply:
Nice!